python/ray/experimental/workflow/storage/base.py

```diff
@@ -39,8 +39,8 @@ class Storage(metaclass=abc.ABCMeta):
     """
 
     @abstractmethod
-    async def load_step_input_metadata(self, workflow_id: str,
-                                       step_id: StepID) -> Dict[str, Any]:
+    async def load_step_input_metadata(self, workflow_id: str, step_id: StepID
+                                       ) -> Awaitable[Dict[str, Any]]:
```

Comment:
sorry, but there may be some misconception: we should not use awaitable for async function return type. here are some official examples: https://github.com/python/cpython/pull/20386/files